packages/components/src/Filename/Filename.tsx

```diff
@@ -0,0 +1,50 @@
+import { forwardRef } from 'react';
+
+import { type TooltipProps as BasicTooltipProps, Tooltip } from '../Tooltip';
+import { type TypographyProps } from '../Typography';
+import { useOverflowed } from '../OverflowTypography/hooks';
+
+import { StyledTypography } from './styles';
+import { useLogic } from './hooks';
+
+type TooltipProps = Omit<BasicTooltipProps, 'ref'>;
+
+type TooltipCustomizable = {
+  /**
+   * Cпособ кастомизировать тултип при необходимости
+   * @example <OverflowTypography tooltipProps={{placement: 'top-start'}} />
+   */
+  tooltipProps?: Omit<TooltipProps, 'children'>;
+};
+
+type FileElementProps = TooltipCustomizable & TypographyProps;
```

Comment:
```suggestion
type FileElementProps = TypographyProps & {
  /**
   * Cпособ кастомизировать тултип при необходимости
   * @example <OverflowTypography tooltipProps={{placement: 'top-start'}} />
   */
  tooltipProps?: Omit<TooltipProps, 'children'>;
};
```

В данному случае нет необходимости создавать тип